{"name": "Verificar que un usuario puede iniciar sesión correctamente con credenciales válidas", "status": "broken", "statusDetails": {"message": "TypeError: Page.get_by_role() takes 2 positional arguments but 3 were given\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features/steps/login_steps.py\", line 15, in step_impl\n    context.login_page.login(Config.VALID_USERNAME, Config.VALID_PASSWORD)\n    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ceciliaperez/Desktop/DGII-Automatizacion/pages/login_page.py\", line 21, in login\n    self.click_element_by_role(\"button\",\"Login\")\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\n  File \"/Users/ceciliaperez/Desktop/DGII-Automatizacion/pages/base_page.py\", line 16, in click_element_by_role\n    self.page.get_by_role(elementType,name).click()\n    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\n"}, "steps": [{"name": "Given que estoy en la página de inicio de sesión", "status": "passed", "start": 1751748011992, "stop": 1751748013714}, {"name": "When introduzco un nombre de usuario y contraseña válidos", "status": "broken", "statusDetails": {"message": "TypeError: Page.get_by_role() takes 2 positional arguments but 3 were given\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features/steps/login_steps.py\", line 15, in step_impl\n    context.login_page.login(Config.VALID_USERNAME, Config.VALID_PASSWORD)\n    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ceciliaperez/Desktop/DGII-Automatizacion/pages/login_page.py\", line 21, in login\n    self.click_element_by_role(\"button\",\"Login\")\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\n  File \"/Users/ceciliaperez/Desktop/DGII-Automatizacion/pages/base_page.py\", line 16, in click_element_by_role\n    self.page.get_by_role(elementType,name).click()\n    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^\n"}, "start": 1751748013714, "stop": 1751748013767}, {"name": "And hago clic en el botón de Login", "status": "skipped", "start": 1751748013887, "stop": 1751748013887}, {"name": "Then debería ser redirigido al tablero con un mensaje de bienvenida \"You logged into a secure area!\"", "status": "skipped", "start": 1751748013887, "stop": 1751748013887}, {"name": "And cierro sesión", "status": "skipped", "start": 1751748013887, "stop": 1751748013887}], "attachments": [{"name": "Failure Screenshot", "source": "46244c20-0fe4-4aec-a2ac-1cfc365b9226-attachment.png", "type": "image/png"}], "start": 1751748011664, "stop": 1751748013887, "uuid": "3f06b62a-5353-4a52-849c-3216f25df073", "historyId": "5e2016984f1b8795171c53a319a2679a", "fullName": "Funcionalidad de Inicio de Sesión de Usuario: Verificar que un usuario puede iniciar sesión correctamente con credenciales válidas", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "login_valid"}, {"name": "feature", "value": "Funcionalidad de Inicio de Sesión de Usuario"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}